### DISCUSSION TITLE
请问一下这个怎么解决

### DISCUSSION THREAD

**Discussion by forchannot** [Mrs4s/go-cqhttp#1535] [Q&A]
![image](https://user-images.githubusercontent.com/100580891/173240575-e070a7ea-78d9-4ee3-a314-3ddbe113ff05.png)
请问下怎么解决，还是说不需要管，rc1没有这个提示

**Comment by pizeroLOL** [comment]
如果功能正常就不用管，如果有问题试试把设备锁开了
